pentagon/component/inventory/__init__.py

```diff
@@ -23,6 +24,12 @@ class Inventory(ComponentBase):
     ]
 
     def __init__(self, data, additional_args=None, **kwargs):
+        # HACK this if is to support start-project workflow
```

Comment:
I think the right thing to do here is to push the required parameters into the aws VPC component. I don't think it needs to be a blocker for this PR but maybe create another issue to do that? In hindsight, I think that's where that requirement should have been in the first place

Reply:
that makes sense - yeah the modules need some love now that there's more support for gcp instead of just aws.

Reply:
https://github.com/reactiveops/pentagon/issues/186